tests/www/test_utils.py

```diff
@@ -16,13 +16,26 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-
+import os
 import unittest
+from datetime import datetime
+
 import mock
 from xml.dom import minidom
 
+from airflow import models, AirflowException
 from airflow.www import utils
+from airflow.www.utils import render_attrs
+
+AIRFLOW_MAIN_FOLDER = os.path.realpath(
```

Comment:
I actually found a simpler way in airflow code: airflow.example_dags.__path__[0] - maybe that approach can be used here and in other places.

Reply:
Thanks. I apply your suggestion.